<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
	<reporter-output>
	</reporter-output>
	<suite name="MFiles Mobile Access - Smoke" duration-ms="24"
		started-at="2016-07-11T12:50:45Z" finished-at="2016-07-11T12:50:45Z">
		<groups>
			<group name="">
				<method
					signature="SmokeTestCases.AdvancedUseCaseTest1_1_10A(java.util.HashMap, java.lang.String)[pri:0, instance:MFMA.Tests.SmokeTestCases@3b6eb2ec]"
					name="AdvancedUseCaseTest1_1_10A" class="MFMA.Tests.SmokeTestCases" />
			</group> <!-- -->
		</groups>
		<test name="Mobile access - Smoke Test cases" duration-ms="24"
			started-at="2016-07-11T12:50:45Z" finished-at="2016-07-11T12:50:45Z">
			<class name="MFMA.Tests.SmokeTestCases">
				<test-method status="PASS"
					signature="onSuiteStart(org.testng.ITestContext)[pri:0, instance:MFMA.Tests.SmokeTestCases@3b6eb2ec]"
					name="onSuiteStart" is-config="true" duration-ms="13"
					started-at="2016-07-11T15:50:45Z" finished-at="2016-07-11T15:50:45Z">
					<params>
						<param index="0">
							<value>
                <![CDATA[org.testng.TestRunner@55f3ddb1]]>
							</value>
						</param>
					</params>
					<reporter-output>
					</reporter-output>
				</test-method> <!-- onSuiteStart -->
				<test-method status="PASS"
					signature="init()[pri:0, instance:MFMA.Tests.SmokeTestCases@3b6eb2ec]"
					name="init" is-config="true" duration-ms="1" started-at="2016-07-11T15:50:45Z"
					finished-at="2016-07-11T15:50:45Z">
					<reporter-output>
					</reporter-output>
				</test-method> <!-- init -->
				<test-method status="FAIL"
					signature="AdvancedUseCaseTest1_1_10A(java.util.HashMap, java.lang.String)[pri:0, instance:MFMA.Tests.SmokeTestCases@3b6eb2ec]"
					name="AdvancedUseCaseTest1_1_10A" duration-ms="0"
					started-at="2016-07-11T15:50:45Z" description="Login to application"
					data-provider="excelDataReader" finished-at="2016-07-11T15:50:45Z">
					<exception class="java.lang.RuntimeException">
						<message>
              <![CDATA[org.testng.SkipException: Test data sheet '' does not exists.]]>
						</message>
						<full-stacktrace>
              <![CDATA[java.lang.RuntimeException: org.testng.SkipException: Test data sheet '' does not exists.
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: org.testng.SkipException: Test data sheet '' does not exists.
	at genericLibrary.DataProviderUtils.excelDataReader(DataProviderUtils.java:95)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:153)
	... 20 more
]]>
						</full-stacktrace>
					</exception> <!-- java.lang.RuntimeException -->
					<reporter-output>
					</reporter-output>
				</test-method> <!-- AdvancedUseCaseTest1_1_10A -->
			</class> <!-- MFMA.Tests.SmokeTestCases -->
		</test> <!-- Mobile access - Smoke Test cases -->
	</suite> <!-- MFiles Mobile Access - Smoke -->
</testng-results>
